package.json

```diff
@@ -1132,7 +1132,7 @@
     "mime": "^2.4.4",
     "mime-types": "^2.1.27",
     "minimatch": "^3.1.2",
-    "moment": "^2.29.4",
+    "moment": "~2.29.4",
```

Comment:
The EBT package brought an update to 2.30.1 and it is incompatible with the date picker format `Aug 01, 2029 @ HH:MM:SS` used in [these tests](https://github.com/elastic/kibana/blob/07a83d6a2849cc166fe00ca356ced7d857fd3c80/x-pack/test/security_solution_cypress/cypress/e2e/explore/urls/state.cy.ts#L63) (and others).

cc @elastic/kibana-visualizations, are you aware of any incompatibilities between the DatePicker and the latest version of `moment`?

Reply:
https://github.com/elastic/kibana/pull/191305 should fix it.